styles/helpers/typography.css

```diff
@@ -96,26 +96,22 @@ body {
   }
 }
 
-
 .heading--light {
```

Comment:
do we want to keep the `--`?

Reply:
I guess we can even remove this since just below we have `.text-color-light` which does exactly the same.